allenact/base_abstractions/sensor.py

```diff
@@ -236,7 +236,7 @@ def __init__(
     ) -> None:
         self.nactions = nactions
         self.expert_args: Dict[str, Any] = expert_args or {}
-
+        observation_space = self._get_observation_space()
```

Comment:
This will change quite a bit with the hierarchical policies PR, but it was missing, yes.

Reply:
Yah, some of my code was complaining so I added this quick band aid, looking forward to your solution being merged :)